### ISSUE TITLE
Robot cannot receive reference message

### ISSUE THREAD

**Issue by hui87667137** [tdlib/td#3179]
I am currently experiencing a problem where I am sending a reference message in a group chat, but the chatbot in the group chat does not receive the reply_to_maessage field in the callback content of the webhook
If it's a super group, this problem doesn't exist
Now I hope the robot can receive the reference content I sent. Is there any solution,
This is my current plan, transferring the group chat to another user will upgrade it to a super group, and then the robots in the super group can receive the reference content I send
Is there any other better solution?
If this is the only solution, how can I more conveniently upgrade to a super group? Because some groups are not owned by me, I cannot do the above operations

**Comment by levlam** [comment]
If the replied message isn't accessible to the bot, then it can't receive the message in private chats and basic groups.

**Comment by hui87667137** [comment]
Yes, the message I quoted is from another robot. So, how can I upgrade to a super group more quickly? Are there any fixed metrics

**Comment by hui87667137** [comment]
thanks

**Comment by Abudl68** [comment]
Why

Great
